packages/cliqz-home-react/src/SpeedDialsRow.js

```diff
@@ -0,0 +1,160 @@
+import React, { Component } from 'react';
+import SpeedDial from './SpeedDial';
+import AddSpeedDial from './AddSpeedDial';
+import cliqz from './Cliqz';
+
+class SpeedDialsRow extends Component {
+  constructor(props) {
+    super(props);
+
+    // this.closeUndo = this.closeUndo.bind(this);
+    this.undoRemoval = this.undoRemoval.bind(this);
+    this.resetAll = this.resetAll.bind(this);
+  }
+
+  componentWillMount() {
+    this.state = {
+      isCustom: this.props.type === 'custom',
```

Comment:
we some times use `isCustom` and sometimes a `type` would be nice if we could clean it up on this occasion